### ISSUE TITLE
Can't download lower resolution videos (240p)

### ISSUE THREAD

**Issue by stewmcl** [meeb/tubesync#162]
I use the get next best resolution option when downloading videos, it fails to fall back to lower than 360p resolutions (in other words it can't seem to grab 240p videos). 
Curious if this can be added/implemented?

**Comment by meeb** [comment]
TubeSync was meant to download content to watch on Plex or Jellyfin or a similar media server so I implemented a hard cut-off at 360p which is why it won't grab 240p. I can make this configurable if there's a request for it.

**Comment by ExodusC** [comment]
Just throwing in my support for allowing the option for downloads of lower resolutions like 240p. It would be a really useful archival feature for old channels.

**Comment by meeb** [comment]
Yeah OK I can see the use case. I'll look for a sensible resolution to permit.
